app/views/shared/_user_info.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= link_to gravatar_for(current_user, size: 50), current_user %>
+<h1><%= current_user.name %></h1>
+<span><%= link_to t(".view_profile"), current_user %></span>
+<span><%= pluralize current_user.microposts.count, t(".micropost") %></span>
```

Comment:
- Ngoài count ra còn có các method: length, size để lấy số lượng bản ghi.
- Sự khác biệt giữa các method này:
  + count: Sử dụng SQL query(SELECT count(*) FROM microposts WHERE...) trong cơ sở dữ liệu
  + length: không count trực tiếp bằng câu lệnh SQL mà load các record từ CSDL rồi tính toán.
  + size: nếu các record đã được load thì đếm số lượng record, nếu các record chưa được load lên thì thực hiện query để count dưới CSDL.